### PR TITLE
Remove team name

### PR DESCRIPTION
**Pull request by ibilal**
Removed teamName from index view, index route, and client src/index. Added teamName as a session variable that pulls the teamName in from the .env config file.

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -6,12 +6,9 @@
   </head>
   <body>
     <h1><%= title %></h1>
+    <input type="hidden" id=teamName value=<%= teamName %>/>
```
Comment (RickyV33) [inline views/index.ejs]: @ibilal, when we tried testing it with a name attribute, was the hidden input included inside the form?

Comment (ibilal) [inline views/index.ejs]: we got an error when adding in the name attribute

**Comment by RickyV33** [comment]
LGTM

File: app.js [hunk 2]
```diff
@@ -30,8 +27,17 @@ app.use(session({
   resave: false,
   saveUninitialized: false
 }));
-app.use(express.static(path.join(__dirname, 'public')));
 
+// Middleware to add the teamName to the session from the .env config file
+app.use(function (req, res, next) {
+  var teamName = req.session.teamName;
+  if (!teamName) {
+    req.session.teamName = process.env.TEAM_NAME || 'sevensource';
```
Comment (kathtran) [inline app.js]: Shouldn't default to `sevensource` -- should perform error handling if the name isn't found

**Pull request merged**